lib/web_ui/lib/src/engine/embedder.dart

```diff
@@ -216,8 +218,12 @@ class FlutterViewEmbedder {
         .instance.semanticsHelper
         .prepareAccessibilityPlaceholder();
 
+    final DomElement announcementsElement = createDomElement('ftl-announcement-host');
+    _accessibilityAnnouncements = AccessibilityAnnouncements(hostElement: announcementsElement);
+
     shadowRoot.append(accessibilityPlaceholder);
     shadowRoot.append(_sceneHostElement!);
+    shadowRoot.append(announcementsElement);
```

Comment:
Do we need announcements on every possible view, or can we have a single global instance?

Reply:
1. Where would we put that instance if it's not `flutterViewEmbedder`? Would that be a separate singleton object? In other words, would you like this to be re-singleton-ified? If so, then the question in the multi-view work is, what can be used to ensure that the live elements end up at/near the end of the DOM? (That being said placing it at the end, this might be slightly less critical after the two other fixes in this PR and the last.)
2. AFAIAA, there's also not a strict issue with having multiple live elements. Each live element is already implemented to support multiple announcements at once.

Reply:
If we can have multiple live elements with no problem, let's go with this, that's why it's LGTM'd :)

(Without wanting to derail this PR: since you said that this needs to be near the end of the DOM, we could put it as the "last child" of the "body" tag. And this wouldn't need to be a re-singletonized: instances would need to either create the DOM object in the DOM if it wasn't there already, or reuse the one created by other instance previously, since all they do is adding some elements to the DOM when asked to announce something.)

Reply:
The challenge is that inserting the live element div at the end of the DOM only works if subsequently initialized DOM elements don't also append themselves to `document.body`, thereby ending up after the live element. Effectively, you'd still need some view-level abstraction that can be used to append more elements to a view without adding them after the accessibility DOM for reliability.